lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Fan.rb

```diff
@@ -0,0 +1,73 @@
+# A variety of fan calculation methods that are the same regardless of fan type.
+# These methods are available to FanConstantVolume, FanOnOff, FanVariableVolume, and FanZoneExhaust
+module ASHRAE901PRMFan
```

Comment:
this module is included in multiple locations.  pretty sure it only needs to be loaded once?

Reply:
@lymereJ : Jeremy, can you look at this one? There are "include ASHRAE901PRMFan" lines in several files. It looks like Weili added those.

Reply:
It looks like we followed the same pattern as the Standard class. The module is defined in `Standards.Fan.rb` and is loaded in `Standards.FanConstantVolume.rb`, `Standards.FanOnOff.rb`, etc. I can see why you would want to remove the module loading from `ashrae_90_1_prm.FanConstantVolume.rb` for example since it's the only thing in it, but why not remove it from `Standards.FanConstantVolume.rb` as well then? Am I missing something?